lib/testing/src/lib/process-services/pages/select-apps-dialog.page.ts

```diff
@@ -0,0 +1,53 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BrowserVisibility } from '../../core/utils/browser-visibility';
+import { by, element, ElementFinder } from 'protractor';
+import { BrowserActions } from '../../core/utils/browser-actions';
+
+export class SelectAppsDialog {
+
+    selectAppsDialog: ElementFinder = element(by.css('mat-dialog-container[aria-labelledby="adf-select-app-dialog-title"]'));
+    title: ElementFinder = element(by.id('adf-select-app-dialog-title'));
+    dropdownAppsButton: ElementFinder = element(by.id('adf-select-app-dialog-dropdown'));
```

Comment:
If this is a dropdown can we use the DropdownPage?

Reply:
done